### ISSUE TITLE
V5 - jsx-a11y/img-has-alt error when using eslint extends airbnb config.

### ISSUE THREAD

**Issue by philipchmalts** [evcohen/eslint-plugin-jsx-a11y#225]
Since img-has-alt has been deprecated, eslint fails even when  ` "jsx-a11y/alt-text": 2,` is added to config because airbnb still has the old rule in its config.

**Comment by evcohen** [comment]
if you are using `eslint-config-airbnb` the version of `eslint-plugin-jsx-a11y` must match the one listed in airbnb's peerDependency config in `package.json` 😞 sorry about that!

**Comment by philipchmalts** [comment]
All good! Thanks for the quick response, went back down to 4.00 for but will post this issue up with them!

**Comment by evcohen** [comment]
anytime! thanks for reporting

**Comment by philipchmalts** [comment]
Looks like they just updated, waiting on a release
